app/src/main/java/com/example/admin/lab_56_57/ActivityForLab57Fragment.java

```diff
@@ -0,0 +1,119 @@
+package com.example.admin.lab_56_57;
+
+import android.app.ActivityManager;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.content.res.Resources;
+import android.graphics.drawable.Drawable;
+import android.support.v4.app.Fragment;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.Spinner;
+
+import com.example.admin.lab_56_57.data.*;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class ActivityForLab57Fragment extends android.support.v4.app.Fragment {
+
+    private AdapterForLinear adapterForLinear_;
```

Comment:
The same thing. We're used to use naming rule: from private to general. So class should be named as LinearAdapter and field as linearAdapter_.
no need to change it right now. But keep it in mind.